dali/test/python/test_RN50_external_source_parallel_data.py

```diff
@@ -0,0 +1,102 @@
+# Copyright (c) 2020, NVIDIA CORPORATION. All rights reserved.
```

Comment:
This file is marked as executable, but doesn't specify the interpreter.

Reply:
Fixed